src/main/java/org/jenkinsci/plugins/tokenmacro/impl/LogRegExMacro.java

```diff
@@ -65,40 +52,26 @@ public String evaluate(AbstractBuild<?, ?> context, TaskListener listener, Strin
 
     @Override
     public String evaluate(Run<?, ?> run, FilePath workspace, TaskListener listener, String macroName) throws MacroEvaluationException, IOException, InterruptedException {
-        return readLogFile(run.getLogFile());
+        return readLogFile(run);
     }
 
-    public String readLogFile(File file) throws IOException {
+    private String readLogFile(Run<?, ?> run) throws IOException {
```

Comment:
Isn't changing the visibility a bit risky?

Reply:
@basil @slide

Reply:
It's a bit risky, yes. I am fine with it though

Reply:
Fine then, thanks!